bin/strata-cli/src/link.rs

```diff
@@ -0,0 +1,184 @@
+use std::fmt::Display;
+
+use alloy::primitives::{Address as StrataAddress, TxHash};
+use bdk_wallet::bitcoin;
+
+/// A thing that exists or is represented onchain.
```

Comment:
```suggestion
/// Represents anything that can be represented on-chain.
```

Reply:
Also, is this valid for Strata as well? Or just bitcoin?
Could be more clear of what `on-chain` means in the docstring.